src/CommunityToolkit.Maui.MediaElement.Analyzers.CodeFixes/CommunityToolkit.Maui.MediaElement.Analyzers.CodeFixes.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Curious why this is not in the existing analyzers project?

Reply:
Not sure I follow, this is also in the non-MediaView analyzers project